### PR TITLE
Sinzi branch

### PR DESCRIPTION
**Pull request by s1nzy**
Here is my stuff

### CODE REVIEW

**Comment by fake-sinzi** [comment]
done for now

File: fake_file.txt [hunk 1]
```diff
@@ -2,7 +2,10 @@ This is my repository
 feature 1
 initial idea 1-1
 initial idea 1-2
+sinzi original idea 1-3
```
Comment (s1nzy) [inline fake_file.txt]:
```suggestion
sinzi awesome idea 1-3
```

File: fake_file.txt [hunk 2]
```diff
@@ -2,7 +2,10 @@ This is my repository
 feature 1
 initial idea 1-1
 initial idea 1-2
+sinzi original idea 1-3
+branch sinzi idea 1-4
```
Comment (s1nzy) [inline fake_file.txt]: nit: capital letter

File: fake_file.txt [hunk 3]
```diff
@@ -2,7 +2,10 @@ This is my repository
 feature 1
 initial idea 1-1
```
Comment (s1nzy) [inline fake_file.txt]: hgft

File: fake_file.txt [hunk 4]
```diff
@@ -2,7 +2,10 @@ This is my repository
 feature 1
 initial idea 1-1
 initial idea 1-2
+sinzi original idea 1-3
+branch sinzi idea 1-4
 
 feature 2
 initial idea 2-1
```
Comment (s1nzy) [inline fake_file.txt]: uh

File: fake_file.txt [hunk 5]
```diff
@@ -2,7 +2,9 @@ This is my repository
 feature 1
 initial idea 1-1
 initial idea 1-2
+sinzi original idea 1-3
```
Comment (fake-sinzi) [inline fake_file.txt]: stick to original idea

File: fake_file.txt [hunk 6]
```diff
@@ -2,7 +2,10 @@ This is my repository
 feature 1
 initial idea 1-1
 initial idea 1-2
+sinzi original idea 1-3
+branch sinzi idea 1-4
 
 feature 2
 initial idea 2-1
 initial idea 2-2
```
Comment (fake-sinzi) [inline fake_file.txt]:
```suggestion
better idea 2-2
```

**Pull request merged**